context-free.md

```diff
@@ -1,22 +1,37 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+People who want to program art, perhaps for interesting geometric shapes.
 
 ## What is easy to do in this language? Why is it easy?
-
+Recursion. We can easily define shapes and recursively create that shape.
 
 ## What is hard to do in this language? Why is it hard?
-
+Art that involves randomness is hard to do. Version 2 does not support 
+function calls and expressions.
 
 ## How did you learn how to program in this language?
-
+Digging around documentation, I found this really helpful
+[page](https://github.com/MtnViewJohn/context-free/wiki/Shape-Adjustments). 
+Looking at the sample code that came with the program was moderately helpful.
+I got help from Sam Andow on understanding loops. Ultimately, what helped me
+the most was and lots and lots of trial and error, which I found to be a 
+pretty frustrating way to learn a language.
```

Comment:
I also agree. I hope that everyone remembers how important documentation and examples are, when you're working on your own languages 😄.